app/views/miq_alert/show.html.haml

```diff
@@ -0,0 +1 @@
+= render :partial => 'alert_details'
```

Comment:
Do we really need the separate partial? It doesn't seem to be shared with anything else...